opam

```diff
@@ -5,9 +5,12 @@ homepage: "https://github.com/ocamllabs/higher"
 bug-reports: "https://github.com/ocamllabs/higher/issues"
 dev-repo: "https://github.com/ocamllabs/higher.git"
 license: "MIT"
-build: [
-  "ocaml" "pkg/build.ml" "native=%{ocaml-native}%"
-                         "native-dynlink=%{ocaml-native-dynlink}%"
+build:
+[[ "ocaml" "pkg/pkg.ml" "build"
+           "--installer" "true" ]]
```

Comment:
Thanks!  Should there be any differences between the `opam` file in the `higher` repository and the `opam` file in `opam-repository`?  e.g. should `"--pinned" "%{pinned}%"` be used in both?